packages/git-semver-tags/test/index.spec.js

```diff
@@ -257,6 +257,24 @@ describe('git-semver-tags', () => {
     })
   })
 
+  it('should handle regexp escaped characters in the tag prefix', function (done) {
+    writeFileSync('test6', '')
+    exec('git add --all && git commit -m"eigth commit"')
+    exec('git tag ms+6.0.0')
+    writeFileSync('test6', '1')
+    exec('git add --all && git commit -m"tenth commit"')
+    exec('git tag ms+7.0.0')
+    writeFileSync('test6', '2')
+    exec('git add --all && git commit -m"eleventh commit"')
+    exec('git tag notms+7.0.0')
+
+    gitSemverTags({ tagPrefix: 'ms+' }, function (err, tags) {
+      if (err) done(err)
+      assert.deepStrictEqual(tags, ['ms+7.0.0', 'ms+6.0.0'])
+      done()
+    })
+  })
+
```

Comment:
Please see how other tests are written. You should use promises, `testTools` and `expect` util.

Reply:
oops, missed the new test format when syncing in the latest master, fixed!